README.md

```diff
@@ -22,235 +22,9 @@ React Native wrapper for the [Spokestack](https://spokestack.io) speech activity
-    "trace-level": Spokestack.TraceLevel.DEBUG // configurable logging level
-  },
-  tts: {
-    'ttsServiceClass': 'io.spokestack.spokestack.tts.SpokestackTTSService',
-    // TTS API account properties. Only set these if you have a Spokestack account.
-    'spokestack-id': 'f0bc990c-e9db-4a0c-a2b1-6a6395a3d97e', // your Spokestack API ID
-    'spokestack-secret': '5BD5483F573D691A15CFA493C1782F451D4BD666E39A9E7B2EBE287E6A72C6B6' // your Spokestack API secret
-  },
-  nlu: {
-    // NLU settings. Only set these if you are calling `Spokestack.classify`.
-    'nlu-model-path': YOUR_NLU_MODEL_PATH, // string filesystem path to nlu model
-    'nlu-metadata-path': YOUR_NLU_METADATA_PATH, // string filesystem path to nlu metadata
-    'wordpiece-vocab-path': YOUR_NLU_VOCABULARY_PATH // string filesystem path to nlu vocab
-  }
-});
-
-// Speech Pipeline
-
-// Start and stop the speech pipeline. All methods can be called repeatedly.
-Spokestack.start(); // start speech pipeline. can only start after initialize is called.
-Spokestack.stop(); // stop speech pipeline
-Spokestack.activate(); // manually activate the speech pipeline. The speech pipeline is now actively listening for speech to recognize.
-Spokestack.deactivate(); // manually deactivate the speech pipeline. The speech pipeline is now passively waiting for an activation trigger.\
-// Binding to speech pipeline events
-const logEvent = e => console.log(e);
-Spokestack.onActivate = logEvent;
-Spokestack.onDeactivate = logEvent;
-Spokestack.onError = e => {
-  Spokestack.stop();
-  logEvent(e);
-};
-Spokestack.onTrace = e => { // subscribe to tracing events according to the trace-level property
-  logEvent(e);
-  console.log(e.message);
-};
-Spokestack.onRecognize = e => {
-  logEvent(e);
-  console.log(e.transcript) // "Hello Spokestack"
-
-// NLU and TTS
-
-// Classify the intent and slot of the transcript
-Spokestack.classify(e.transcript, {})
-// Get a URL to a real-time synthesize
-Spokestack.synthesize({'input': e.transcript, 'format': Spokestack.TTSFormat.TEXT, 'voice': 'demo-male'})
-};
-// Receive the transcript classifcation result
-Spokestack.onClassification = e => {
-  logEvent(JSON.stringify(e))
-  console.log(e.result.intent)
-}
-// Receive the real-time transcript synthesis result
-Spokestack.onSuccess = e => {
-  logEvent(JSON.stringify(e))
-  console.log(e.url) // https://api.spokestack.io/stream/g2dkABVnYXRld2F5QDE3Mi4yNy4xMi4yNDQAACeUAAAAAgE
-};
-
-```
+Take fifteen minutes to [get started using Spokestack](https://www.spokestack.io/docs/React%20Native/getting-started), or check out our in-depth tutorials on [ASR](https://www.spokestack.io/docs/React%20Native/speech-pipeline), [NLU](https://www.spokestack.io/docs/React%20Native/nlu), and [TTS](https://www.spokestack.io/docs/React%20Native/tts). Also be sure to take a look at the [Cookbook](https://www.spokestack.io/docs/React%20Native/cookbook) for quick solutions to common problems!
```

Comment:
"Take fifteen minutes" sounds aggressive to me. Maybe just [Get started using Spokestack]...